### PR TITLE
[Improvement-13911][JSON] Improve the readability of workflow exported json files

### LINKED ISSUE #13911: [Improvement][JSON] Improve the readability of workflow exported json files
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

* Improve the readability of workflow exported json files

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rickchengx**
## Purpose of the pull request

* close: #13911 
* Improve the readability of workflow exported json files

## Brief change log

* before
截屏2023-04-11 20 09 02

* after
截屏2023-04-11 20 08 44

## Verify this pull request
manually tested
截屏2023-04-11 20 09 20

### CODE REVIEW

**Review by SbloodyS — APPROVED** [review]
LGTM if CI passed

**Review by tungmeoo — APPROVED** [review]
yêu cầu kéo

**Review by EricGao888 — APPROVED** [review]

**Comment by EricGao888** [comment]
I've restarted failed CI job. Waiting for CI to pass.

**Pull request merged**
